utilities/class.tapestry-helpers.php

```diff
@@ -219,6 +219,12 @@ public static function userIsAllowed($action, $nodeMetaId, $tapestryPostId, $sup
         $groupIds = self::getGroupIdsOfUser($userId, $tapestryPostId);
         $user = new TapestryUser($userId);
 
+        // If user is author and node is in review, user is not allowed to edit node
+        $inReview = isset($node->reviewStatus);
```

Comment:
This is not accurate. Only nodes with an `accepted` or `submitted` review status are not allowed to be edited by their author. Nodes that are in review but have a review status of `rejected` should still be editable by the original author.

Reply:
Thanks for catching that. Changed!